www/base/src/app/builders/builder/builder.tpl.jade

```diff
@@ -20,6 +20,7 @@
                 span(title="{{br.submitted_at | dateformat:'LLL'}}")
                   | {{br.submitted_at | timeago }}
               td
+                span(ng-if="br.properties.owners === undefined") {{br.properties.owner[0]}}
```

Comment:
I wonder if it makes sense to use "owners" by default and migrate build request to also use that naming long term.